app/uk/gov/hmrc/fhregistrationfrontend/views/confirm_delete.scala.html

```diff
@@ -16,37 +16,36 @@
 
 @import org.joda.time.DateTime
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
-@this(
-    mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
-)
+@this(layout: layout, viewHelpers: ViewHelpers)
 
 @(timeSaved: DateTime)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@title = @{
+@pageTitle = @{
   Messages("fh.delete_confirmation_page.header")
 }
 
-@mainTemplate(title = title) {
+@layout(pageTitle) {
 
-  <form action="@{uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.deleteUserData}" method="POST">
+    @viewHelpers.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.deleteUserData, 'novalidate -> "novalidate") {
```

Comment:
we are adding this only at those p[laces where it was there already . If it was not there, i think no need to add that